content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -422,6 +439,15 @@ However, configuring the proxy this way does require cluster provider specific k
 In addition to that, you loose monitoring of access to external services and cannot apply Istio features on traffic to
 external services.
 
+The third approach directs the traffic through the Istio sidecar proxy, however it allows access to any service on any
+ports that have no HTTP service inside the mesh and no service entry defined. With this approach, like with the
+previous one, you cannot monitor access to external services. The advantage of this approach is that you don't need to
+know which IP ranges are external to the cluster. Another advantage is that you can easily switch to the first approach
+for a specific port by simply creating a service entry for that port. This way you can start Istio by allowing access
+to any external service and then decide to start controlling access to external services for specific ports, enabling
+monitoring and traffic control features later, once they are needed. A caveat is that some ports, for example port 80,
+already have HTTP services inside Istio by default, so for them only the first and the second approaches can be used.
```

Comment:
```suggestion
already have HTTP services inside Istio by default. Because of this caveat, you can only use the first and second approaches for those ports.
```